### PR TITLE
Update CHANGELOG for the release

### PR DESCRIPTION
**Pull request by sharidas**
Update CHANGELOG for the release

Signed-off-by: Sujith H <sharidasan@owncloud.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 - Remove trailing comma - [#46](https://github.com/owncloud/impersonate/issues/46)
 
+## [0.1.1] - 2017-11-07
+
+### Changed
+
+- Add dist to Makefile - [#65](https://github.com/owncloud/impersonate/issues/65)
+- Change screenshot, fix indentation - [#61](https://github.com/owncloud/impersonate/issues/61)
+- Add summary, bugtracker, repo, docs, change description - [#60](https://github.com/owncloud/impersonate/issues/60)
+- Update the CHANGELOG file - [#51](https://github.com/owncloud/impersonate/issues/51)
```
Comment (PVince81) [inline CHANGELOG.md]: please only leave things that make sense for the end-user

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+
+### Fixed
+
+- Prevent further level impersonation - [#63](https://github.com/owncloud/impersonate/issues/63)
+- Clear session when impersonation doesnt happen - [#62](https://github.com/owncloud/impersonate/issues/62)
+- Restrict impersonate to admin user from subadmin user - [#49](https://github.com/owncloud/impersonate/issues/49)
+- Remove trailing comma - [#46](https://github.com/owncloud/impersonate/issues/46)
```
Comment (PVince81) [inline CHANGELOG.md]: remove or "minor code cleanup"

File: CHANGELOG.md (hunk 1 above)
Comment (sharidas) [reply CHANGELOG.md]: sure

**Comment by sharidas** [comment]
Updated the CHANGELOG.

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+
+### Changed
+
+- Replace Phony target all with dist in Makefile - [#65](https://github.com/owncloud/impersonate/issues/65)
+- Change screenshot, fix indentation - [#61](https://github.com/owncloud/impersonate/issues/61)
+
+### Fixed
+
+- Prevent further level impersonation - [#63](https://github.com/owncloud/impersonate/issues/63)
+- Clear session when impersonation doesnt happen - [#62](https://github.com/owncloud/impersonate/issues/62)
```
Comment (PVince81) [inline CHANGELOG.md]: doesn't happen, missing quote. Or use "does not"

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,6 +10,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Restrict impersonate to admin user from subadmin user - [#49](https://github.com/owncloud/impersonate/issues/49)
 - Remove trailing comma - [#46](https://github.com/owncloud/impersonate/issues/46)
 
+## [0.1.1] - 2017-11-07
+
+### Changed
+
+- Replace Phony target all with dist in Makefile - [#65](https://github.com/owncloud/impersonate/issues/65)
+- Change screenshot, fix indentation - [#61](https://github.com/owncloud/impersonate/issues/61)
```
Comment (PVince81) [inline CHANGELOG.md]: maybe append "in app info"

**Comment by sharidas** [comment]
Updated.

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,8 +7,23 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## [Unreleased]
 ### Changed
 
+- Use precompiled handlebars for templates in javascript files - [#70](https://github.com/owncloud/impersonate/pull/70)
```
Comment (PVince81) [inline CHANGELOG.md]: please don't add these, we haven't even merged them yet

Comment (sharidas) [reply CHANGELOG.md]: Ok.

**Comment by sharidas** [comment]
Updated.

**Comment by PVince81** [comment]
@SergioBertolinSG @patrickjahns not sure now whether this PR should be against the master branch or release branch. @patrickjahns will clarify

**Comment by PVince81** [comment]
according to the process, we'll merge the release branch back into master after the release is done, so no need for this PR

**Pull request closed without merging**